android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -526,6 +528,22 @@ public boolean handleMessage(Message msg)
 				Activity activity = getWindowActivity();
 				if (activity != null) {
 					activity.setTitle(TiConvert.toString((Object) (msg.obj), ""));
+
+					if (windowActivity != null && windowActivity.get() != null
+						&& windowActivity.get().getSupportActionBar() != null) {
+						ActionBar actionBar = windowActivity.get().getSupportActionBar();
+						if (actionBar.getTitle() instanceof SpannableStringBuilder) {
+							SpannableStringBuilder stringBuilder;
+							stringBuilder = new SpannableStringBuilder(TiConvert.toString((Object) (msg.obj), ""));
```

Comment:
Is there a reason why the constructor is split from the assignment?

Reply:
you are right, can be one command (still two lines otherwise it will complain about the length :) )